rust/psibase/src/time.rs

```diff
@@ -72,32 +73,48 @@ impl<'de> Deserialize<'de> for TimePointSec {
 }
 
 #[derive(
-    Debug,
-    Copy,
-    Clone,
-    Default,
-    PartialEq,
-    Eq,
-    PartialOrd,
-    Ord,
-    Pack,
-    Unpack,
-    ToKey,
-    ToSchema,
-    SimpleObject,
-    InputObject,
+    Debug, Copy, Clone, Default, PartialEq, Eq, PartialOrd, Ord, Pack, Unpack, ToKey, ToSchema,
 )]
 #[fracpack(
     definition_will_not_change,
     fracpack_mod = "fracpack",
     custom = "TimePointUSec"
 )]
 #[to_key(psibase_mod = "crate")]
-#[graphql(input_name = "TimePointUSecInput")]
 pub struct TimePointUSec {
     pub microseconds: i64,
 }
 
+impl TryFrom<String> for TimePointUSec {
```

Comment:
Shouldn't this be `FromStr`?

Reply:
It's not quite clear to me why it should be one over the other. I guess `FromStr` is the more specific tool for the job...

Reply:
Oh, for std::str::parse compatibility?

Reply:
Done

Reply:
I would use `From<String>` only for types that adopt the string directly into their state. `FromStr` is better when the string is parsed.